examples/express-react/server/server.js

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import express from "express";
+import cors from "cors";
+import { load } from "@azure/app-configuration-provider";
+import { ConfigurationMapFeatureFlagProvider, FeatureManager } from "@microsoft/feature-management";
+
+const app = express();
+const port = 5000;
+
+const corsOptions = {
+    origin: ["http://localhost:4173", "http://localhost:5173"] // vite will run the web app on port 4173 and 5173
```

Comment:
why this? does `vite` has anything to do with the example?

Reply:
Or you regard `server` and `client` as a single react example instead of standalone? If so, please leave something in `express-react/README.md` which is now empty.

Reply:
It's still not finished. Let me turn it to draft PR.